include/wx/generic/busyinfo.h

```diff
@@ -37,12 +37,15 @@ class WXDLLIMPEXP_CORE wxBusyInfo : public wxObject
         Init(wxBusyInfoFlags().Parent(parent).Label(message));
     }
 
+    void UpdateText(const wxString& str, bool is_markup = true);
+
     virtual ~wxBusyInfo();
 
 private:
     void Init(const wxBusyInfoFlags& flags);
 
     wxFrame *m_InfoFrame;
+    wxControl *m_Text;
```

Comment:
You need to add forward declaration of `wxControl` to fix the CI builds.

Also, even though the variable above uses a wrong naming convention, for some reason, let's still follow the convention for this one and call it `m_text`.